client/src/views/Mapping/components/InputColumns/index.tsx

```diff
@@ -92,22 +70,26 @@ const InputColumns = ({ schema, selectedAttribute, source }: IProps) => {
                           return (
                             <div className="stacked-tags">
                               <Tag>SCRIPT</Tag>
-                              <StringSelect
-                                disabled={true}
-                                inputItem={
-                                  attribute && attribute.mergingScript
-                                    ? attribute.mergingScript
-                                    : ""
-                                }
-                                items={["mergingScript.py"]}
+                              <ScriptSelect
                                 loading={loading}
-                                onChange={(e: string) => {
+                                selectedScript={attribute.mergingScript}
+                                onChange={(script: string) => {
```

Comment:
I don't think you need to have a block function here: `onChange={ (script:string) => updateAttribute({...}) }`

Reply:
Not sure I follow. What would you write instead?

Reply:
the `updateAttribute` function does not need to be in a block, it can be used as return value. it's not very important though.